pkgs/development/python-modules/mailrise/default.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  apprise,
+  aiosmtpd,
+  pyaml,
+}:
+buildPythonPackage rec {
+  pname = "mailrise";
+  version = "1.4.0";
+  dependencies = [
+    apprise
+    aiosmtpd
+    pyaml
+  ];
+  src = fetchFromGitHub {
+    owner = "YoRyan";
+    repo = "mailrise";
+    tag = version;
+    hash = "sha256-QgeGDQeSsfvopgBgQQsWrx036SX1FhE67LI8M8rJM/Q=";
+  };
```

```suggestion
  version = "1.4.0";

  src = fetchFromGitHub {
    owner = "YoRyan";
    repo = "mailrise";
    tag = version;
    hash = "sha256-QgeGDQeSsfvopgBgQQsWrx036SX1FhE67LI8M8rJM/Q=";
  };

  dependencies = [
    apprise
    aiosmtpd
    pyaml
  ];

```
we should also set build-system